tsk/img/aff.c

```diff
@@ -216,7 +216,8 @@ aff_close(TSK_IMG_INFO * img_info)
 {
     IMG_AFF_INFO *aff_info = (IMG_AFF_INFO *) img_info;
     af_close(aff_info->af_file);
-	for (int i = 0; i < img_info->num_img; i++) {
+	int i;
```

Comment:
Can you move this up to line 217 because other compilers are going to complain that it isn't defined at the top block.

Reply:
Fixed